x-pack/plugins/alerting/server/task_runner/action_scheduler/schedulers/per_alert_action_scheduler.test.ts

```diff
@@ -175,7 +175,8 @@ describe('Per-Alert Action Scheduler', () => {
     test('should generate executable for each alert and each action', async () => {
       const scheduler = new PerAlertActionScheduler(getSchedulerContext());
       const executables = await scheduler.generateExecutables({
-        alerts,
+        activeCurrentAlerts: alerts,
```

Comment:
can we add a unit test for an active alert without a scheduled action to ensure no executable gets generated?